src/transforms/operations.js

```diff
@@ -387,7 +387,7 @@ export function setSelectionOperation(transform, properties) {
   }
 
   // Resolve the selection keys into paths.
-  if (props.anchorKey) {
+  /* if (props.anchorKey) {
```

Comment:
I've commented this part since the `getPath` are failing because we are resolving a removed key in the new document. `getPath` throws `Could not find a descendant node with key "ia36".`.

But I'm not removing it till I understand what is the exact reason you were using paths instead of keys.